lra/lra-coordinator/README.adoc

```diff
@@ -1,75 +1,56 @@
-`LOG_LEVEL`::
-  It could be defined on docker run. The value sets log level for messages produced by the LRA coordinator.
-  The permitted values are log levels defined by Quarkus runtime (https://quarkus.io/guides/logging).
-  The default log level value is `INFO`.
-`OBJECT_STORE_PATH`::
-  It could be defined on docker run.
-  It defines a directory where the transaction log is stored. Transaction log is the persistent storage
-  of the LRA coordinator. The default location is directory `/data`.
+* `NARAYANA_VERSION` is used only during the `build` phase and defines what version of Narayana's LRA coordinator  will be downloaded
+* `NARAYANA_FULL_URL` is used during the `build` phase to set up the location where the Narayana distro will be downloaded from
+* `LOG_LEVEL` is used during the `run` phase to set up the level/accuracy of logging produced by the LRA coordinator. The permitted values are logging levels defined in the [Quarkus runtime](https://quarkus.io/guides/logging). The default logging level value is `INFO`.
+* `OBJECT_STORE_PATH` is used during the `run` phase to set up a directory where the transaction log is stored (which is the persistent storage of the LRA coordinator). The default location is the directory `/data`.
 
-The lra coordinator is run under user `jboss` and exposes port `8080` for communication.
+The lra coordinator is run by the user `jboss` and exposes port `8080` for communication.
 
-== Docker build
+== Build
 
 ```bash
 export NARAYANA_VERSION=...
-docker build -t lra-coordinator --build-arg NARAYANA_VERSION=$NARAYANA_VERSION .
+podman build --tag lra-coordinator --build-arg NARAYANA_VERSION=${NARAYANA_VERSION}
 ```
 
-== Running localy
+== Running the image locally
 
-I you want to run locally you use the following command. This starts the docker
-image `lra-cordinator` you've just built and define the logging to be set to
-TRACE level.
+The following command runs `lra-cordinator` locally and sets up the logging level to `TRACE`.
 
 ```bash
-docker run -p 8080:8080 --rm -it -e LOG_LEVEL=TRACE lra-coordinator
+podman run -p 8080:8080 --rm -it -e LOG_LEVEL=TRACE lra-coordinator
 ```
 
-== Docker image at Dockerhub
+== Image Repository
 
-The `lra-coordinator` image is available at https://hub.docker.com/r/jbosstm/lra-coordinator.
-You can pull it (the tag `latest`) by running
+The image of `lra-coordinator` is available in Narayana's official [quay.io repository](https://quay.io/repository/jbosstm/lra-coordinator):
 
 ```bash
-docker pull docker.io/jbosstm/lra-coordinator
+podman pull quay.io/jbosstm/lra-coordinator:latest
 ```
 
-== Release the lra docker image to Dockerhub
+== Release the lra docker image
 
-If you build the image you can push the new version to dockerhub with sequence
+After building the image of `lra-coordinator`, it is possible to tag and push the new version to `quay.io` following the commands:
```

Comment:
Add a comment showing how to build the image:
  `podman build --tag lra-coordinator --build-arg NARAYANA_VERSION=${NARAYANA_VERSION} .`

Reply:
Added a reference to the `Build` section.